saw-core-coq/coq/Makefile

```diff
@@ -1,13 +1,23 @@
-all: build
+all: Makefile.coq
 
 Makefile.coq: _CoqProject
 	coq_makefile -f $< -o $@
 
-build: Makefile.coq
-	make -f Makefile.coq
+include Makefile.coq
```

Comment:
TIL: Make will try to make the target of an include and only fail if the target doesn't exist after this attempt.  Nice!

Reply:
Thanks! :) It took me a while to come up with this trick, but it allows dependencies to be added to those specified by the Makefile.coq.